ConfigSpace/hyperparameters/beta_integer.pxd

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+cimport numpy as np
+np.import_array()
+
+# We now need to fix a datatype for our arrays. I've used the variable
+# DTYPE for this, which is assigned to the usual NumPy runtime
+# type info object.
+DTYPE = float
+# "ctypedef" assigns a corresponding compile-time type to DTYPE_t. For
+# every type in the numpy module there's a corresponding compile-time
+# type with a _t-suffix.
+ctypedef np.float_t DTYPE_t
+
+from .uniform_integer cimport UniformIntegerHyperparameter
```

Comment:
I'm a bit cautious with relative imports. If it works, then no problem, but I would try to make it consistent across all of them.

Reply:
Yes, will fix this with the next push.